cwms-data-api/src/test/java/cwms/cda/api/CatalogControllerTestIT.java

```diff
@@ -445,4 +442,133 @@ void test_loc_kind() {
         ;
     }
 
+    @Test
+    void testFilterLocations() throws Exception{
+        String officeId = "SPK";
+        String json = loadResourceAsString("cwms/cda/api/valid_base_location.json");
+        Location location = new Location.Builder(Formats.parseContent(Formats.parseHeader(Formats.JSON, Location.class),
+            json, Location.class))
+            .withOfficeId(officeId)
+            .build();
+        String baseLocationName = location.getName();
+
+        TestAccounts.KeyUser user = TestAccounts.KeyUser.SPK_NORMAL;
+
+        // create base location
+        createLocation(baseLocationName, true, officeId, location.getLocationKind());
```

Comment:
can we not pass an entire Location DTO into this function? Like don't add it if not, I'm just surprised.